### PR TITLE
feat: add cache-from major-minor-branch

### PR DESCRIPTION
**Pull request by innovaker**
Lets any branch utilize its corresponding major-minor-branch cache.

See: #62

### CODE REVIEW

**Comment by innovaker** [comment]
I'm not a fan of the `major-minor-branch` term, but I'm yet to come up with something better.  Suggestions more than welcomed!

**Review by petejohanson — COMMENTED** [review]
One thought on naming.

File: .github/workflows/containers.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ jobs:
       candidate: ${{ steps.definitions.outputs.candidate }}
       versions: ${{ steps.definitions.outputs.versions }}
       major-minor: ${{ steps.definitions.outputs.major-minor }}
+      major-minor-branch: ${{ steps.definitions.outputs.major-minor-branch }}
```
[the remark this answers is not in the document]
Comment (innovaker) [reply .github/workflows/containers.yml:46]: This is an excellent point I hadn't considered.  Thanks!

[the remark this answers is not in the document]
Comment (innovaker) [reply .github/workflows/containers.yml:46]: I woke up:
- `major-minor-trunk`
  - effectively means the same as `main`/`master` but doesn't crowd the existing namespace.
- `x-y-branch`
  - would require remaining `MAJOR_MINOR` to `X_Y` throughout the code, which I personally interpret as a stunned face emoji.

I still don't love them, but they're food for thought.

If we lack better ideas, I suggest moving forward with `major-minor-trunk` for now.

[the remark this answers is not in the document]
Comment (petejohanson) [reply .github/workflows/containers.yml:46]: I slept on this one too considering. I'm fine w/ leaving as is, or going w/ something like `major-minor-branch-suffix` to be clear it's just the suffix. That being said, that's probably needlessly noisy.

How about we leave as is, and pepper with a comment or two to make sure it's clear where used?

[the remark this answers is not in the document]
Comment (innovaker) [reply .github/workflows/containers.yml]: I've added some examples.

**Review by petejohanson — APPROVED** [review]
Thanks!

**Pull request merged**